source/audio/soundSplit.py

```diff
@@ -85,15 +100,15 @@ def initialize() -> None:
 			log.exception("Could not initialize audio session manager")
 			return
 		state = SoundSplitState(config.conf["audio"]["soundSplitState"])
-		setSoundSplitState(state)
+		setSoundSplitState(state, initial=True)
 	else:
 		log.debug("Cannot initialize sound split as WASAPI is disabled")
 
 
 @atexit.register
 def terminate():
 	if nvwave.usingWasapiWavePlayer():
-		setSoundSplitState(SoundSplitState.OFF)
+		setSoundSplitState(SoundSplitState.NVDA_BOTH_APPS_BOTH)

```

Comment:
Why this change? This reads to me as sound split being set to both channels on every terminate, effectively meaning that parameters are still being changed when terminating.

Reply:
Fixed that.